Lib/test/libregrtest/cmdline.py

```diff
@@ -161,7 +162,7 @@ def __init__(self, **kwargs) -> None:
         self.trace = False
         self.coverdir = 'coverage'
         self.runleaks = False
-        self.huntrleaks = False
+        self.huntrleaks: tuple[int, int, str] | Literal[False] = False
```

Comment:
Why not use None as default?

Reply:
That would be cleaner! Wasn't sure if you'd consider that a controversial change or not, since it's changing working code rather than just changing type annotations :-)

Reply:
Done in [ad6b96a](https://github.com/python/cpython/pull/109405/commits/ad6b96ae001601689b5a24f6ad68fc7d42604a27)